src/config.js

```diff
@@ -40,10 +40,8 @@ export const APPSIGNAL_APP_ENV = env('APPSIGNAL_APP_ENV', 'development');
 
 // CRON jobs
 export const CRON_SCHEDULE = env('CRON_SCHEDULE', '0 * * * *');
-export const DOCUMENT_CRON_UPDATE_PERIOD_DAYS = env('DOCUMENT_CRON_UPDATE_PERIOD_DAYS', 7);
-export const DOCUMENT_CRON_CREATED_AGE_DAYS = env('DOCUMENT_CRON_CREATED_AGE_DAYS', undefined);
-export const DOCUMENT_CRON_REFRESH_ENABLED = env('DOCUMENT_CRON_REFRESH_ENABLED', true);
-export const DOCUMENT_CRON_UNEDITED_DAYS = env('DOCUMENT_CRON_UNEDITED_DAYS', 30);
+export const DOCUMENT_CRON_UPDATE_PERIOD = env('DOCUMENT_CRON_UPDATE_PERIOD', 604800000);
+export const DOCUMENT_CRON_STALE_PERIOD = env('DOCUMENT_CRON_STALE_PERIOD', 2592000000);
```

Comment:
It’s good to multiply out the values here so it’s more easily editable/readable.  Not a big deal, but it’s handy to avoid bugs etc., since you can’t really tell what values like `604800000` would meaningfully translate to just by looking at it.

E.g.

```js

export const DOCUMENT_CRON_UPDATE_PERIOD = env('DOCUMENT_CRON_UPDATE_PERIOD',  someNumberOfDays * 24 * 60 * 60 * 1000);
```